restructure_by_scope.py

```diff
@@ -0,0 +1,56 @@
+
+"""
+Restructures CredData files into scope-based folders (/src/, /test/, etc.)
+"""
+import os
+import shutil
+from pathlib import Path
+
+# From CredSweeper's ml_config.json
+SCOPES = [
+    "/src/",
+    "/test/",
+    "/conf/",
+    "/script/",
+    "/dist-packages/",
+    "/site-packages/",
+    "/example/",
+    "/record/",
+    "/tool/",
+    "/usr/",
+    "/assets/"
+]
+
+def get_scope(file_path: str) -> str:
```

Comment:
the function might be used in download_data.py instead https://github.com/Samsung/CredData/blob/5046987ceef9916c9afbacf721103942d3c99104/download_data.py#L26